camera_service/camera_service.py

```diff
@@ -0,0 +1,122 @@
+import os
+import asyncio
+import numpy as np
+from caproto.server import ioc_arg_parser, run, pvproperty, PVGroup
+import simulacrum
+import zmq
+import time
+from zmq.asyncio import Context
+import pickle
+
+class ProfMonService(simulacrum.Service):
+    default_image_size = 1024*1024
+
+    def __init__(self):
+        print('Initializing PVs') 
+        super().__init__()
+
+        #load Profmon properties from file
+        with open('screenProps5.dat', 'rb') as file_handle:
```

Comment:
This will break if you try to run the service from outside the camera_service/ directory.  Right now we *try* to always run the services inside their home directories, but I don't want to guarantee it.  Look at model_service.py to see how to make sure the file path is always relative to the python file (as opposed to the directory where the interpreter launched).

Reply:
Good catch! Let me fix that.